buildenv/jenkins/jobs/infrastructure/copyrightCheck

```diff
@@ -61,30 +68,313 @@ timeout(time: 6, unit: 'HOURS') {
                         REGEX = "\'\\(c\\) Copyright IBM Corp. ([0-9]{4}), ${DATE_YEAR} All Rights Reserved\'"
                     } else {
                         echo "ERROR: Unrecognized repository. Unable to determine correct Copyright regex"
-                        sh 'exit 1'
+                        currentBuild.result = 'FAILURE'
```

Comment:
Sorry, I misled you on this one. Setting the build to FAILURE will not actually stop the build. Reccomend removing this line and converting the previous line from an echo into an error.
```suggestion
                        error("Unrecognized repository. Unable to determine correct Copyright regex")
```

Reply:
Done